### DISCUSSION TITLE
huge error in all things you like to edit in V5.5.7

### DISCUSSION THREAD

**Discussion by jayjayvm** [hassio-addons/addon-vscode#621] [Q&A]
The error is unable to load schemas from 'http://schemas.home-assistant.io/integrations......' no content.

There is already a discussion started in the forum on how to solve this and people are rolling back the latest version to V5.5.6.
So it seems there is a fault in the new code.

**Comment by frenck** [comment]
This is not the issue tracker. Also, you could have checked the issue tracker for answers.

TL;DR, this is not an add-on issue.

**Reply by jayjayvm** [reply]
Sorry Frenck,

Not very well known with the protocol on this, next time I will! Just wanted to make people aware here of the issue at hand, and seen the forum I am not the only one with this problem.
